Patch in the correct compiler, flags, and Makefile verbosity, since there is no
other evident way to do this.
https://github.com/timlegge/perl-IO-Compress-Brotli/issues/5
--- Makefile.PL.orig	2023-10-30 19:35:30.000000000 -0500
+++ Makefile.PL	2024-03-27 20:52:50.000000000 -0500
@@ -18,6 +18,11 @@
     }
 }
 
+$cmake_options .= ' -DCMAKE_C_COMPILER="@CC@"';
+$cmake_options .= ' -DCMAKE_C_FLAGS="@CFLAGS@"';
+$cmake_options .= ' -DCMAKE_EXE_LINKER_FLAGS="@LDFLAGS@"';
+$cmake_options .= ' -DCMAKE_VERBOSE_MAKEFILE=ON';
+
 WriteMakefile(
 	NAME             => 'IO::Compress::Brotli',
 	VERSION_FROM     => 'lib/IO/Compress/Brotli.pm',
